api/editor/api-editor/bin/pom.xml

```diff
@@ -11,7 +11,7 @@
 	</parent>
 	<groupId>ca.bc.gov.tno</groupId>
 	<artifactId>api-editor</artifactId>
-	<version>0.0.1-alpha</version>
+	<version>0.0.1-SNAPSHOT</version>
```

Comment:
Maven has a syntax that treats SNAPSHOT like a temporary build.  This enables easier development testing.